library/core/class.applicationmanager.php

```diff
@@ -118,6 +118,7 @@ public function checkApplication($applicationName) {
      * @return bool|mixed Returns the application's info, a specific value, or false if the application cannot be found.
      */
     public function getApplicationInfo($applicationName, $key = null) {
+        $applicationName = ucfirst($applicationName);
```

Comment:
I think this is too risky. Maybe look where the version is being added and add the ucfirst before this call instead.